wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -511,6 +493,47 @@ public TypeSpec generateMessage(MessageType type) {
     return builder.build();
   }
 
+  /**
+   * Preallocate all of the names we'll need for this type. Names are allocated in precedence
```

Comment:
technically, should be `for {@code type}` instead of "for this type". not a big deal